src/libs/ReportUtils.js

```diff
@@ -52,13 +53,16 @@ Onyx.connect({
     },
 });
 
-let allReports;
+let allReports = {};
```

Comment:
you have also added `if (!allReports)` condition in other place, so you don't need to assign empty object here.

Reply:
Nice !!